Tests/web3swiftTests/remoteTests/InfuraTests.swift

```diff
@@ -11,6 +11,12 @@ import Web3Core
 // MARK: Works only with network connection
 class InfuraTests: XCTestCase {
 
+    func testGetNetwork() async throws {
+        let requestURLstring = "https://" + Networks.Mainnet.name + Constants.infuraHttpScheme + Constants.infuraToken
+        let web3 = try await Web3HttpProvider(url: URL(string: requestURLstring)!, network: nil)
+        XCTAssertEqual(web3.network?.chainID, 1)
+    }
+
```

Comment:
Let's create a dedicated `Web3HttpProvider` test file.
This is more than just `getNetwork` test.